docs/source/user-guide/tutorials/building-conda-packages.rst

```diff
@@ -375,42 +362,43 @@ To build a GDAL package:
 
 #. Install Visual Studio:
 
-   * For Python 3, install `Visual Studio 2017 <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2017>`_.
-     Choose Custom install. Under Programming Languages, select workloads that
-     come from Visual Studio so you choose the Desktop Development with C++ and
+   * For Python 3, install `Visual Studio 2017`_.
```

Comment:
This bulleted list needs to be broken up a bit more to make sense. Right now, the Python 3 and Python 2 instructions are all on the same level. Maybe lines 366-367 and 371-373 should be indented again.